src/test/java/org/cloudfoundry/promregator/endpoint/InvalidateCacheEndpointTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.cloudfoundry.promregator.endpoint
+
+import io.mockk.called
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.cloudfoundry.promregator.cfaccessor.CFAccessorCache
+import org.cloudfoundry.promregator.scanner.CachingTargetResolver
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus
+
+//@SpringBootTest(classes = MockedAppInstanceScannerEndpointSpringApplication.class)
+class InvalidateCacheEndpointTest {
+    private lateinit var subject: InvalidateCacheEndpoint
+    private val cfAccessorCache = mockk<CFAccessorCache>(relaxed = true)
+    private val targetResolver = mockk<CachingTargetResolver>(relaxed = true)
```

Comment:
We can keep using Mockito for mocking in tests, but mockk brings a 'strict' mocking mode (along with some other great features). 

When a mock is called that is strict, if a stub hasn't been created, then we get an exception (with useful info about what was called, so you can easily mock it out). With Mockito, instead you get surprising null pointers.

In this test since we are just verifying, we are OK with the relaxed mode.